en/flight_controller/intel_aero.md

```diff
@@ -67,19 +67,28 @@ After setting up the PX4 development environment, follow these steps update the
 
 ## Connecting a Lidar Lite range finder
 
-The following instructions are for a Lidar Lite V3 connected via I2C. The I2C port on the Aero (labled compass) is used for the external magnetometer (part of the GPS). Therefore a I2C splitter has to be used to connect the Lidar Lite (see picture).
-
-![](../../assets/hardware/Aero_I2C_splitter.JPG)
+The following instructions are for a Lidar Lite V3 connected via I2C. The I2C port on the Aero (labled compass) is used for the external magnetometer (part of the GPS). Therefore, either a I2C splitter has to be used to connect the Lidar Lite (see picture) or the Telemetry port can be used (recommended as it is not being used). In addition it is recommended to use a electrolytic capacitor to reduce spikes in the distance readings (see [here](https://static.garmin.com/pumac/LIDAR_Lite_v3_Operation_Manual_and_Technical_Specifications.pdf) on page 3).
 
 The pinout for the Lidar Lite V3 is as follows
 
-| pin | Aerofc I2C | Lidar Lite V3    |
-| --- | ---------- | ---------------- |
-| 1   | VCC        | VCC              |
-| 2   | SCL        | - (Power enable) |
-| 3   | SDA        | - (Mode control) |
-| 4   | GND        | SCL              |
-| 5   | -          | SDA              |
-| 6   | -          | GND              |
+| pin | Aerofc TELEMETRY | Lidar Lite V3    |
+| --- | ---------------- | ---------------- |
+| 1   | VCC              | VCC              |
+| 2   | TX               | - (Power enable) |
+| 3   | RX               | - (Mode control) |
```

Comment:
connecting 'mode control' to RX port could cause it change to PWM mode maybe is better advise user to leave this unconnected.

Reply:
Ah no, - should mean that it is not being used.